department-of-veterans-affairs/abd-vro
# Scan packages for dto-classes specified in conf-camel.yml

## What was the problem?


Had to specify each Java DTO (data transfer object) class so that Camel can serialize and deserialize for RabbitMQ messages.

## How does this fix it?


Allows us to specify the package name. All classes in that package will be registered for automatic Camel serialization and deserialization.

## How to test this PR
See this in the `app` image's log:

```
2022-10-19 19:20:49.624  INFO 1 --- [           main] o.a.c.impl.engine.AbstractCamelContext   : Apache Camel 3.19.0 (camel-1) started in 3s616ms (build:217ms init:1s432ms start:1s967ms)
2022-10-19 19:20:49.743  INFO 1 --- [           main] gov.va.vro.camel.CamelConfiguration      : Registering CamelDtoConverter for: class gov.va.vro.service.spi.model.ClaimMetricsInfo
2022-10-19 19:20:49.760  INFO 1 --- [           main] gov.va.vro.camel.CamelConfiguration      : Registering CamelDtoConverter for: class gov.va.vro.service.spi.model.GeneratePdfPayload
2022-10-19 19:20:49.761  INFO 1 --- [           main] gov.va.vro.camel.CamelConfiguration      : Registering CamelDtoConverter for: class gov.va.vro.service.spi.model.Claim
2022-10-19 19:20:49.764  INFO 1 --- [           main] gov.va.vro.camel.CamelConfiguration      : Registering CamelDtoConverter for: class gov.va.vro.model.mas.MasAutomatedClaimPayload
2022-10-19 19:20:49.764  INFO 1 --- [           main] gov.va.vro.camel.CamelConfiguration      : Registering CamelDtoConverter for: class gov.va.vro.model.mas.ClaimCondition
2022-10-19 19:20:49.765  INFO 1 --- [           main] gov.va.vro.camel.CamelConfiguration      : Registering CamelDtoConverter for: class gov.va.vro.model.mas.VeteranIdentifiers
2022-10-19 19:20:49.765  INFO 1 --- [           main] gov.va.vro.camel.CamelConfiguration      : Registering CamelDtoConverter for: class gov.va.vro.model.mas.MasExamOrderStatusPayload
2022-10-19 19:20:49.766  INFO 1 --- [           main] gov.va.vro.camel.CamelConfiguration      : Registering CamelDtoConverter for: class gov.va.vro.model.mas.ClaimDetail
2022-10-19 19:20:49.796  INFO 1 --- [           main] gov.va.vro.camel.CamelConfiguration      : 8 endpoints: 
...
```
